core/kt-osrd-utils/src/main/kotlin/fr/sncf/osrd/utils/units/Duration.kt

```diff
@@ -0,0 +1,31 @@
+package fr.sncf.osrd.utils.units
+
+import kotlin.math.absoluteValue
+
+@JvmInline
+value class Duration(val milliseconds: Long) : Comparable<Duration> {
+
+    val absoluteValue get() = Duration(milliseconds.absoluteValue)
+    val seconds get() = milliseconds / 1000.0
+
+    operator fun plus(value: Duration): Duration {
+        return Duration(milliseconds + value.milliseconds)
+    }
+
+    operator fun minus(value: Duration): Duration {
+        return Duration(milliseconds - value.milliseconds)
+    }
+
+    companion object {
+        @JvmStatic
+        val ZERO = Duration(milliseconds = 0L)
+        fun fromSeconds(seconds: Double) = Duration(milliseconds = (seconds * 1_000.0).toLong())
```

Comment:
```suggestion
        @JvmStatic
        @JvmName("fromSeconds")
        fun fromSeconds(time: Double): Duration = time.seconds
        @JvmStatic
        @JvmName("toSeconds")
        fun fromSeconds(time: Duration): Double = time.seconds
```

These methods are meant to help work with Durations from java